README.md

```diff
@@ -1,43 +1,34 @@
 # Discovery Service
 
-
-Comparison with *discovery.etcd.io* package:
-- backward compatible
-
-
-
-# Usage
-
+The `discovery.etcd.io` package is only usable during the initial bootstrap of the etcd cluster. We extended this package to support for dynamic maintenance of
+cluster membership while keeping it backward-compatible with the original package.
 
 
 # APIs
 
-- `GET /cluster-id/`: returns a list of all members of the cluster with given cluster-id
-- `POST /renew`:
-
-# Discussion
-
-- This implementation provides security by obscurity.
-
-# References
-This project extends (forks) the discovery.etcd.io's project.
+- adding `members/CLUSTER_TOKEN` which returns a comma-separated list of *latest* members of the cluster.
```

Comment:
i think the API format you had originally was more understandable to state the route first, then a description of it after